src/app/page.tsx

```diff
@@ -1,40 +1,7 @@
 import React from 'react';
 
+import UIPREVIEW from '@/components/UIPreview';
```

Comment:
We should keep it as with the file name naming convention, `UIPREVIEW ` -> `UIPreview`, although could be better naming with `UiPreview`

Reply:
Good catch. I missed it, my bad. Resolved. :raised_hands: